### PR TITLE
disk storage updates

### PR DESCRIPTION
**Pull request by nahern**
updates to satisfy audit ticket https://jira.iohk.io/browse/CAD-1944

### CODE REVIEW

File: doc/getting-started/building-the-node-using-nix.md [hunk 1]
```diff
@@ -31,6 +31,24 @@ git clone https://github.com/input-output-hk/cardano-node
...
+
+trusted-substituters = 
+
+trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= 
+
+max-jobs = 2 # run at most two builds at once 
+
+cores = 0 # the builder will use all available CPU cores 
+
+nix
```
Comment (dcoutts) [inline doc/getting-started/building-the-node-using-nix.md:47]: What is this line? Is this supposed to be within the `nix.conf`?

File: doc/getting-started/building-the-node-using-nix.md [hunk 2]
```diff
@@ -31,6 +31,24 @@ git clone https://github.com/input-output-hk/cardano-node
...
+
+Once the node is compiled, you should run the clean up command:
+collect-garbage
```
Comment (dcoutts) [inline doc/getting-started/building-the-node-using-nix.md:50]: This is currently inside the code block from the config file above which is not intended. The command to run should be in a code block on its own.

Are you sure the command is ```collect-garbage``` and not ```nix collect-garbage```? Perhaps that's what's going on with the stray `nix` above. Can you double check.

**Review by dcoutts — CHANGES_REQUESTED** [review]
I think it is unnecessary (and a little patronising) to explain the consequences of running out of disk space. It's sufficient to say that building itself will take considerable disk space, perhaps more than even the current size of the blockchain, but it's only temporary and can be cleaned up afterwards. Then specifically with nix, here is the command to run to clean up afterwards.

Then in addition to reclaiming disk space after the build is complete, there's also a nice sample nix config to use to get much quicker builds by reusing cached binaries from IOHK. This involves trusting IOHK as a source for binary packages. This point needs to be made clear since it's a trust/security issue.

File: doc/getting-started/building-the-node-using-nix.md (hunk 2 above)
Comment (nahern) [reply doc/getting-started/building-the-node-using-nix.md:50]: thanks Duncan, will check that with DevOps.

File: doc/getting-started/building-the-node-using-nix.md (hunk 1 above)
Comment (nahern) [reply doc/getting-started/building-the-node-using-nix.md:47]: thanks for that feedback, will reword the para and check the inline snippet.

File: doc/getting-started/building-the-node-using-nix.md (hunk 2 above)
Comment (nahern) [reply doc/getting-started/building-the-node-using-nix.md:50]: Sam clarified that the command is nix-collect-garbage

File: doc/getting-started/building-the-node-using-nix.md (hunk 1 above)
Comment (nahern) [reply doc/getting-started/building-the-node-using-nix.md:47]: the example has that line within nix.conf, should it be removed?

**Review by nahern — COMMENTED** [review]
command updated and comments implemented

File: doc/getting-started/building-the-node-using-nix.md (hunk 1 above)
Comment (dcoutts) [reply doc/getting-started/building-the-node-using-nix.md:47]: I think that must have been a mistake or copy'n'paste problem or something. I don't think it's valid in a nix config file.

File: doc/getting-started/building-the-node-using-nix.md [hunk 3]
```diff
@@ -32,6 +32,27 @@ cd cardano-node
...
+cores = 0 # the builder will use all available CPU cores 
+
+nix
+
+```
+
+Once the node is compiled, you should run the clean up command:
+`nix-collect-garbage`
+
+In addition, there is a sample nix config that you can use which reuses cached binaries from IOHK and results in faster builds. This involves trusting IOHK as a source for binary packages.
```
Comment (dcoutts) [inline doc/getting-started/building-the-node-using-nix.md:54]: Right, but the nix config in question is the one above! So if you want to do it in this order, move the config example after this sentence.

**Comment by Jimbo4350** [comment]
Closing this. If this is still relevant please re-open a new PR.

**Pull request closed without merging**